build/buildBuildImages.sh

```diff
@@ -16,6 +16,7 @@ source $REPO_DIR/build/__phpVersions.sh    # For PHP_BUILD_BASE_TAG
 source $REPO_DIR/build/__nodeVersions.sh   # For YARN_CACHE_BASE_TAG
 source $REPO_DIR/build/__nodeVersions.sh   # For YARN_CACHE_BASE_TAG
 source $REPO_DIR/build/__sdkStorageConstants.sh
+source $REPO_DIR/build/__buildBuildImageFunctions.sh
```

Comment:
Some of these functions use variables like `BUILD_SIGNED` in them which is set actually later than this line, so they might fail.